tests/serverpod_test_server/generated/migration/migrations/serverpod_test/20231128092552086/migration.sql

```diff
@@ -326,6 +326,324 @@ CREATE TABLE "unique_data" (
 -- Indexes
 CREATE UNIQUE INDEX "email_index_idx" ON "unique_data" USING btree ("email");
 
```

Comment:
Question: Did you make the change to add the full SQL code in this PR?

Reply:
If so, did you generate all migrations for all projects (since they would now be responsible for migrating all tables in their migration)

Reply:
I did not add that in this PR as that affects the apply and we have to change the folder structure for that. Coming in next PR.

Reply:
Hmm, but in this file we have added a bunch of lines? That should not have happened if that was the case.